libchisel/src/lib.rs

```diff
@@ -3,7 +3,7 @@ extern crate binaryen;
 extern crate parity_wasm;
 extern crate rustc_hex;
 
-use parity_wasm::elements::Module;
+pub use parity_wasm::elements::Module;
```

Comment:
Not needed yet, but perhaps we could benefit from a prelude of sorts containing re-exports and the like.